.changeset/dry-rivers-wink.md

```diff
@@ -0,0 +1,14 @@
+---
+"@paprika/textarea": major
+"@paprika/table": minor
+"@paprika/inline-editing": patch
+"@paprika/list-box": patch
+"@paprika/seducer": patch
+---
+
+- 📦 `new` Introducing: inline-editing in alpha version ready to start integrating it but still needs work
+- 💥 Breaking Change for Textarea removing an awkward way for declaring a ref in the textarea
+- ⚠️ Remove keyboard navigation from the table for now, meanwhile we decided which is the best rout for this, we can bring it back later.
+  this is not a breaking change because no body is using it!
+- 🙋‍♀️ Table component now support
+- 🔨 Upgraded collapsibleCard on the documentation
```

Comment:
You could have up to 5 different files (so each could have its own release notes) if you wanted